library_generation/model/generation_config.py

```diff
@@ -62,6 +63,17 @@ def get_proto_path_to_library_name(self) -> dict[str, str]:
     def is_monorepo(self) -> bool:
         return len(self.libraries) > 1
 
+    def __validate(self) -> None:
+        seen_library_names = dict()
+        for library in self.libraries:
+            library_name = library.get_library_name()
+            if library_name in seen_library_names:
+                raise ValueError(
+                    f"{library.name_pretty} has the same library name with "
```

Comment:
Can we make the error message more verbose and actionable? Something like
```
f"Both {library.name_pretty} and {seen_library_names.get(library_name)} have the same library_name: 
{library_name}, please update one of the library to have a different library_name."
```

Reply:
Another edge case is that two libraries may have the same name_pretty, but I think the error message is already good enough, so we may not have to consider it.

Reply:
Done.